src/components/Dashboard/Header/WelcomeView.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import welcomeView from '../../assets/welcomeView.svg';
+
+export default class WelcomeView extends Component {
+  render() {
+    return (
+      <>
```

Comment:
`React.Fragment` is used to render two relating tags without ever needing to wrap it into any container. This component only renders an image and `React.Fragment` here is needless.

https://reactjs.org/docs/fragments.html